topics/metabolomics/tutorials/gc_ms_with_xcms/tutorial.md

```diff
@@ -0,0 +1,482 @@
+
+# Retention index calculation
+
+The retention index ({% cite van1963generalization %}) is a way how to convert equipment- and experiment-specific retention times into system-independent normalised constants for GC-based experiments. The retention index of a compound is computed from the retention time by interpolating between the retention times adjacent alkanes which are assigned a fixed retention index. This can be different for the individual chromatographic system, but the derived retention indices are independent and allow comparing values measured by different analytical laboratories.
+
+We use the package {% tool [RIAssigner](toolshed.g2.bx.psu.edu/repos/recetox/riassigner/riassigner/0.3.2+galaxy1) %} to compute retention indices for files in the `.msp` format using an indexed reference list of alkanes in `.csv` or `.msp` format. The output follows the same format as the input but with added retention index values. These can be used at a subsequent stage to improve compound identification ({% cite kumari2011applying %}). Multiple computation methods (e.g. piecewise-linear or cubic spline) are supported by the tool.
+
+> <hands-on-title> Retention index calculation </hands-on-title>
+>
+> 1. {% tool [RIAssigner](toolshed.g2.bx.psu.edu/repos/recetox/riassigner/riassigner/0.3.2+galaxy1) %} with the following parameters:
+>    - In *"Query dataset"*:
+>        - {% icon param-file %} *"Query compound list"*: `Mass spectra from RAMClustR` (output of **RAMClustR** {% icon tool %})
+>    - In *"Reference dataset"*:
+>        - {% icon param-file %} *"Reference compound list"*: `reference_alkanes.csv` (Input dataset)
+>
+>    You can leave the other parameters with their default values.
+>
+>    > <comment-title> Minutes vs. seconds </comment-title>
+>    >
+>    > You might notice that in the last **XCMS** step, we converted the retention times to minutes. However, in this step, we are using seconds again. The reason is that **RAMClustR** converted them internally again to seconds. Regarding the reference compound list, this database already has its retention times in seconds.
+>    {: .comment}
+>
+>    > <details-title> Kováts method </details-title>
+>    >
+>    > The Kováts retention index (or Kováts index) of a compound is its retention time normalized to the retention times of adjacently eluting n-alkanes. It is based on the fact that the logarithm of the retention time converges to the number of carbon atoms in the alkane. For an isothermal chromatogram, you use the following equation to calculate the Kováts index: 
+>    > 
+>    > $$I = 100 \times \frac{\mathtt{log}~t_x - \mathtt{log}~t_n}{\mathtt{log}~t_{n+1} - \mathtt{log}~t_n}$$ 
```

Comment:
I have encountered a problem with rendering this mathematical expression. It occurs when used in a line block inside the details block of the hands-on section. The math expression is displayed before the details block is expanded.

Collapsed details block:
![collapsed](https://user-images.githubusercontent.com/15349569/225601131-2dd94103-551a-483b-ad47-ef188b36311d.png)

Expanded details block:
![expanded](https://user-images.githubusercontent.com/15349569/225601147-94933ba7-ee84-4af8-9ffb-59aa204f89ca.png)

I tried to find something similar in previous materials, it seems like [here](https://github.com/galaxyproject/training-material/blob/d13771e996e13fdc821a28556692a8e4e1675291/topics/single-cell/tutorials/scrna-raceid/tutorial.md?plain=1#L429) someone had the same issue. But there, it is rendering fine, so maybe this only occurs when running the GTN page in Gitpod?

Reply:
Ahhh interesting. What an odd corner case. Ok, I will look into it. Thanks for alerting us!